### DISCUSSION TITLE
Issue with remote build - Java Gradle

### DISCUSSION THREAD

**Discussion by gopac25** [GoogleCloudPlatform/buildpacks#308] [General]
I am trying to build sample application code and ended with this.

[builder] === Java - Gradle (google.java.gradle@0.9.0) ===
[builder] --------------------------------------------------------------------------------
[builder] Running "./gradlew clean assemble -x test --build-cache --quiet"
[builder] Done "./gradlew clean assemble -x test --build-cache --quiet" (716.8?s)
[builder] Failure: (ID: 14f2a5b3) executing command "./gradlew clean assemble -x test --build-cache --quiet": fork/exec ./gradlew: permission denied
[builder] --------------------------------------------------------------------------------
[builder] Sorry your project couldn't be built.
[builder] Our documentation explains ways to configure Buildpacks to better recognise your project:
[builder]  -> https://cloud.google.com/docs/buildpacks/overview
[builder] If you think you've found an issue, please report it:
[builder]  -> https://github.com/GoogleCloudPlatform/buildpacks/issues/new
[builder] --------------------------------------------------------------------------------
[builder] ERROR: failed to build: exit status 1
ERROR: failed to build: executing lifecycle. This may be the result of using an untrusted builder: failed with status code: 51
ERROR
ERROR: build step 0 "gcr.io/k8s-skaffold/pack" failed: step exited with non-zero status: 1
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

BUILD FAILURE: Build step failure: build step 0 "gcr.io/k8s-skaffold/pack" failed: step exited with non-zero status: 1
ERROR: (gcloud.builds.submit) build d954578f-0828-4b2f-97b4-7e820920003a completed with status "FAILURE"

**Comment by jama22** [comment]
@gopac25  would it be possible to share which sample app you tried to use? Also, how were you building this app, using `pack` CLI, Cloud Run?

**Reply by gopac25** [reply]
Yes. I am using below.

git clone https://github.com/GoogleCloudPlatform/buildpack-samples.git
cd sample-java-gradle

gcloud builds submit --pack image=europe-docker.pkg.dev/<project>/<Repo>/sample

**Reply by jama22** [reply]
Just checking back in on this. I tried deploying the project with a few configurations, and they all seemed to work:

* building locally with `pack`
* building to a bucket in `us-central1`
* building to a bucket in `europe-west2`
* building to a multi-region `europe` bucket

They all seemed to work. 

Is there any more info you could provide us @gopac25 ? Did you modify the source code locally, or maybe run gradle commands locally before submitting?  Maybe some other Java experts can weigh in here
